src/mmw/mmw/settings/production.py

```diff
@@ -34,6 +34,7 @@
 # EMAIL CONFIGURATION
 EMAIL_BACKEND = 'apps.core.mail.backends.boto.EmailBackend'
 EMAIL_BOTO_CHECK_QUOTA = False
+DEFAULT_FROM_EMAIL = 'ADD-REAL-ADDRESS-HERE'
```

Comment:
We can default this to something like `noreply@mmw.azavea.com` for now. The `mmw.azavea.com` domain will be useful in testing the Amazon SES DKIM setup.